css/styleexperience.css

```diff
@@ -0,0 +1,72 @@
+img.logo{
+	width: 600px;
+	height: 150px;
+	margin-left: auto;
+	margin-right: auto;
+	display: block;
+	margin-bottom: 30px;
+	margin-top: -40px;
+}
+
+a{
+	color: inherit;
+	text-decoration: none;
+}
+
+a:hover{
+	color:#fff44f;
+	-webkit-text-stroke: 1px black;
+}
+
+
+hr{
+	height: 10px;
+	background-color: #E5E4E2;
+	color: #E5E4E2;
+	border: none;
+
+}
+
+nav{
+	font-family: "tw cen mt";
+	text-align: center;
+	font-size: 150%;
+	text-transform: uppercase;
+	margin-top: 15px;
+	margin-bottom: 15px;
+	margin-left: -20px;
+}
+
+img.lemon{
+	width: 300px;
```

Comment:
I'm not actually seeing this indentation in my code. Maybe I can show you in class?